besu/src/test/java/org/hyperledger/besu/RunnerTest.java

```diff
@@ -159,7 +159,7 @@ public void fastSyncFromGenesis() throws Exception {
 
   private void syncFromGenesis(final SyncMode mode, final GenesisConfigFile genesisConfig)
       throws Exception {
-    final Path dataDirAhead = temp.newFolder().toPath();
+    final Path dataDirAhead = Files.createTempDirectory(temp, "db-ahead");
```

Comment:
I thought @TempDir created the dir automatically?

Reply:
it does but for runnerbuilder we create the database dir and we also need the parent dir